main.go

```diff
@@ -516,6 +516,7 @@ func mainCore() error {
 
 		r.Mount("/explorer", explore.Mux)
 		r.Get("/blocks", explore.Blocks)
+		r.Get("/windows", explore.StakeDiffWindows)
```

Comment:
Same thing with "/windows" needing to be something else like "/pos-intervals" unless we intend to have /windows describe all kinds of Decred windows/intervals.

Reply:
The window is for both PoS and PoW. What other kinds of windows are we considering? Voting activation? I dont think those would get confused here, but in any case will still need a simple url if /windows is not to be used.  /diffwindows ? The url should also match the page title which is still set to Windows.

Reply:
As I pointed out in (my other comment)[https://github.com/decred/dcrdata/pull/750#discussion_r227439237], they need not be the same even though they currently are.   But I can assume they will always be for this PR.

However, another windows I am talking about is the block reward.  The subsidy windows/intervals.